Convert metric aggs docs runtime fields (#71260)
elastic/elasticsearch: docs/reference/aggregations/metrics/median-absolute-deviation-aggregation.asciidoc

==== Script

> In our example above, product reviews are on a scale of one to five. If we
> wanted to modify them to a scale of one to ten, we can use a <<runtime,runtime field>>.

Reviewer: ```suggestion
In the example above, product reviews are on a scale of one to five. If you
want to modify them to a scale of one to ten, use a <<runtime,runtime field>>.
```
